content/en/getting_started/application_security/vulnerability_management.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Getting Started with Application Vulnerability Management
+kind: documentation
+further_reading:
+- link: "/security/application_security/terms"
+  tag: "Documentation"
+  text: "Application Security terms and concepts"
+- link: "/security/application_security/how-appsec-works"
+  tag: "Documentation"
+  text: "How Application Security Management works"
+- link: "/security/application_security/getting_started"
+  tag: "Documentation"
+  text: "Enabling ASM"
+- link: "https://securitylabs.datadoghq.com/"
+  tag: "Security Labs"
+  text: "Security research, reports, tips, and videos from Datadog"
+---
+
+
+## Overview
+
+Datadog [Application Vulnerability Management][1] continuously monitors your production environment for vulnerabilities in the open source libraries your applications rely on. You can identify and prioritize the remediation of the highest vulnerabilities by business impact.
+
+This guide walks you through best practices for getting your team up and running with AVM.
+
+## Phase 1:
+1. Check [ASM Compatibility][2] to see if your service is supported.
+2. Enable Application Vulnerability Management on your services. 
+   - Navigate to [**Security -> Configuration -> Application Security -> Quick Start Guide**][4].
+   - Expand **Enable Vulnerability Detection**.
+   - Click **Start Activation**.
+   - Choose services to secure with ASM.
+## Phase 2: 
+1. **Identify Vulnerabilities**: Navigate to [**Security -> Application Security -> Vulnerabilities**][4].  
+2. Sort by `Status` and `Severity`:
+
+ {{< img src="getting_started/appsec/AVM_Vuln_Severity_Status.png" alt="Application Vulnerability Management showing the Vulnerability tab, sorted by Status and Severity." style="width:100%;" >}}
```

Comment:
```suggestion
   {{< img src="getting_started/appsec/AVM_Vuln_Severity_Status.png" alt="Application Vulnerability Management showing the Vulnerability tab, sorted by Status and Severity." style="width:100%;" >}}
```
Indenting it 3 spaces aligns it so that it looks like its part for step 2. You'll want to do this with the next paragraph and table too.